src/import/FSHTank.ts

```diff
@@ -22,20 +22,33 @@ export class FSHTank {
   }
 
   /**
-   * Finds the profile in the tank by name, if it exists
-   * @param {string} name - The name of the profile we're looking for
+   * Finds the profile in the tank by name, id, or alias, if it exists
+   * @param {string} key - The name or id of the profile we're looking for
    * @returns {Profile | undefined}
    */
-  public findProfileByName(name: string): Profile | undefined {
-    return this.getAllProfiles().find(profile => profile.name === name);
+  public findProfile(key: string): Profile | undefined {
+    return this.getAllProfiles().find(p => p.name === key || p.id === key);
```

Comment:
For consistency, should this also check for a match on URL?  I know URL isn't a field on the object, but since you have the config, you could easily construct what the URL _will_ be and check against that.